tools/build-jetpack.sh

```diff
@@ -70,7 +70,7 @@ hash yarn 2>/dev/null || {
     exit 1;
 }
 yarn --cwd $TARGET_DIR cache clean
-COMPOSER_MIRROR_PATH_REPOS=1 yarn --cwd $TARGET_DIR run build
+yarn --cwd $TARGET_DIR run build-production
```

Comment:
Why remove the `COMPOSER_MIRROR_PATH_REPOS=1`?

Reply:
It's already part of `yarn build-production` (which fires `yarn build-production-php` https://github.com/Automattic/jetpack/blob/master/package.json#L31